scripts/start.js

```diff
@@ -179,4 +194,10 @@ const hot = process.env.SKU_HOT !== 'false';
 
     openBrowser(url);
   });
+
+  onDeath(() => {
+    devServer.stopCallback(() => {
+      debug('Webpack dev server closed');
+    });
+  });
```

Comment:
Just found https://webpack.js.org/configuration/dev-server/#devserversetupexitsignals, might replace this and the same bit in `start-ssr` with that config instead.